### DISCUSSION TITLE
Better information density

### DISCUSSION THREAD

**Discussion by orbisvicis** [spacecowboy/Feeder#126] [Ideas]
The images in card view are too large; I can only fully see 2 news entries.

Compact view is too compact. I can fully see 10 items but the images are too small to make out and the lack of spacing makes everything jumbled.

Based on other RSS readers I believe the ideal layout fits 5-6 news entries.

**Comment by Tunsinasro** [comment]
You can set article style to compact and set the max number of lines to 5, see the screenshot below
![Screenshot_2023-12-20-15-22-57-212_com nononsenseapps feeder play](https://github.com/spacecowboy/Feeder/assets/147873384/f262f73d-85e0-4b79-bc98-89a074d4db4a)
